### ISSUE TITLE
Create design for main URL shortener page

### ISSUE THREAD

**Issue by samrose3** [samrose3/teeny-timey-url#2]
## Description

Currently, the form to shorten a URL is not styled. The main page needs a layout and some style.

## As-Is 
| Initial | After creating URL |
| ------- | ------------------ |
| ![screen shot 2019-02-25 at 2 26 10 pm](https://user-images.githubusercontent.com/11774595/53366526-6bf1ac00-3909-11e9-8090-651d8d7962ac.png) | ![screen shot 2019-02-25 at 2 26 23 pm](https://user-images.githubusercontent.com/11774595/53366535-714ef680-3909-11e9-8da4-d3f64d05906a.png) |

**Comment by bstee615** [comment]
I'd like to take this task. Do you have any aversion to CSS transpilers such as SASS?

**Comment by samrose3** [comment]
@bstee615 Nice, that would be great. I was actually going to add SCSS eventually, but feel free to make the addition!

https://github.com/zeit/next-plugins/tree/master/packages/next-sass

Could you provide designs before implementing? Don't want you to put too much effort into the code before agreeing on a layout and style 😅 

## Features to consider

- Section to house a local history of generated URLs (History panel)
- Additional input to allow a user to make the URL protected by a passcode

I have another friend also creating screens for this, but I would love to see what you come up with as well! More ideas the better.

Cheers

**Comment by samrose3** [comment]
@bstee615 share any design specs in this Issue if you are able to work on this 👍

**Comment by bstee615** [comment]
Yes, I mocked up a screen for it in vanilla HTML and CSS. I will link an image of it and a gist with its current (not-finished) state.

**Mockup**
![image](https://user-images.githubusercontent.com/22305849/53380687-416d1680-393c-11e9-875e-4146669b2ca2.png)

**[Link to Gist](https://gist.github.com/bstee615/8bff53d617634ee551ef16c1d820d6b1)**

**Comment by samrose3** [comment]
@bstee615 solid start.

I like the sidepanel layout for the History. Also, nice job incorporating the passcode feature.

I was imagining the passcode to be visible to the user in the history panel. That way, if a user forgets what the passcode was for a shared URL, they can come back to the history panel to find it.

Good work 👏

**Comment by bstee615** [comment]
@samrose3 Ohhh, that makes a lot more sense. I don’t be know what I was thinking.

So, you want the passcode so that when someone goes to the shortened URL they need a passcode to continue to the destination? Either way, while the feature is not there I will not implement UI for it in this PR.

Should I proceed with implementing and pushing the new UI? I plan to also work on the History feature, but I will put it into another commit (and make an issue to track it if you like).
